internal/services/redis/redis_cache_resource_test.go

```diff
@@ -143,6 +143,7 @@ func TestAccRedisCache_BackupDisabled(t *testing.T) {
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
+			ExpectNonEmptyPlan: true,
```

Comment:
doing this would signify there's a bug to fix?

Reply:
Yes, there is a [bug](https://github.com/Azure/azure-rest-api-specs/issues/3037). I have added a comment to it.